app/src/main/java/org/go/sopt/winey/util/fragment/WineyDialogFragment.kt

```diff
@@ -36,17 +40,47 @@ class WineyDialogFragment(
         binding.btnDialogNegative.text = negativeButtonLabel
     }
 
-    private fun initNegativeButtonClickListener(handleNegativeButton: () -> Unit) {
+    private fun initNegativeButtonClickListener(handleNegativeButton: (() -> Unit)?) {
         binding.btnDialogNegative.setOnClickListener {
-            handleNegativeButton.invoke()
+            handleNegativeButton?.invoke()
             dismiss()
         }
     }
 
-    private fun initPositiveButtonClickListener(handlePositiveButton: () -> Unit) {
+    private fun initPositiveButtonClickListener(handlePositiveButton: (() -> Unit)?) {
         binding.btnDialogPositive.setOnClickListener {
-            handlePositiveButton.invoke()
+            handlePositiveButton?.invoke()
             dismiss()
         }
     }
+
+    companion object {
+        private const val ARG_TITLE = "title"
+        private const val ARG_SUB_TITLE = "subTitle"
+        private const val ARG_NEGATIVE_BUTTON_LABEL = "negativeButtonLabel"
+        private const val ARG_POSITIVE_BUTTON_LABEL = "positiveButtonLabel"
+
+        fun newInstance(
+            title: String,
+            subTitle: String,
+            negativeButtonLabel: String,
+            positiveButtonLabel: String,
+            handleNegativeButton: () -> Unit,
+            handlePositiveButton: () -> Unit
+        ): WineyDialogFragment {
+            val args = Bundle()
+            args.putString(ARG_TITLE, title)
+            args.putString(ARG_SUB_TITLE, subTitle)
+            args.putString(ARG_NEGATIVE_BUTTON_LABEL, negativeButtonLabel)
+            args.putString(ARG_POSITIVE_BUTTON_LABEL, positiveButtonLabel)
```

Comment:
apply 이용하면 조금은 더 깔끔해질 수 있을 것 같아요 ~!

```suggestion
        ): WineyDialogFragment {
            val args = Bundle().apply{
                    putString(ARG_TITLE, title)
                    putString(ARG_SUB_TITLE, subTitle)
                    putString(ARG_NEGATIVE_BUTTON_LABEL, negativeButtonLabel)
                    putString(ARG_POSITIVE_BUTTON_LABEL, positiveButtonLabel)
          }
```